Dispatchfile

```diff
@@ -22,7 +22,7 @@ dindTask("dispatch-integration-test",
                  command = ["make","dispatch-test"],
                  workingDir = "/workspace/src-git",
                  env = [k8s.corev1.EnvVar(name = "DISPATCH_CI", value = "true"),
-		        k8s.corev1.EnvVar(name = "AWS_REGION", value= "us-west-2"),
+		                k8s.corev1.EnvVar(name = "AWS_REGION", value= "us-west-2"),
```

Comment:
I would expect this to be intended as much as the line below it, not more 🤔 . Can you confirm?

Reply:
Applied pep8 to fix the overall formatting. I think there might have been a tab in there somehow.